### ISSUE TITLE
MSFS2024: events.txt not loaded completely if filesize exceeds the size stated in layout.json

### ISSUE THREAD

**Issue by felixc7** [MobiFlight/MobiFlight-Connector#1859]
**Describe the bug**
The events.txt file, when generated from MF connector (as of 21 Nov 2024) is 1,336 KB. In MSFS2024, the MF event module will only load 353 KB of the file, since this file size is specified in layout.json. Any event definition after the first 353 KB of the file will not be loaded. MSFS2020 did not seem to process the file size stated in layout.json in the same way, making the file size statement in the .json file irrelevant and preventing any issues.

To fix this bug, I can see a few options: 
- The layout.json could be updated whenever events.txt changes in size. This seems a little cumbersome..
- ~~The stated file size of events.txt in layout.json can be set to a higher number (e.g. 10,000 KB). This seems like a bit of a botch..~~ Does not work!
- Specify a larger file size (e.g. 10,000 KB), fill up file with some form of padding. This seems like even more of a botch.

**Steps To Reproduce**
Debug-compile the WASM module and scroll down the MobiFlight messages in the MSFS2024 console. It will load about 3800 Event definitions. The last definition is truncated. 
Alternatively: Try using one of the AS530_xxx events that appear late in the events.txt, they will not work. Move it to an earlier position in the file and it will work.

**Comment by vanderhaas** [comment]
I'll chime in.
The events (and events.user) files are useful to trigger mobiflight events from Flight Tracker Stream Deck plugin. Very useful tool for those with compact setups.
Both events and events.user files will not work properly in MSFS2024 if they don't have exact size set in layout.json

For me it's not a huge issue. I can edit it myself with every change.

**Comment by vanderhaas** [comment]
@DocMoebiuz yes. I sometimes have a slight issue with the last event on the list not working, but it might be due to the wrong size being reported by windows or something like that. Other than that it works fine when json matches the actual file size.
